### ISSUE TITLE
Surveys: Once survey taken do not allow edit only copy

### ISSUE THREAD

**Issue by paulbert** [open-learning-exchange/planet#5091]
EDIT: You can ignore this comment, it's not really relevant to the direction we decided on.

We will need to have a better way to handle survey versions for the CSV export.  Right now the export uses the latest version of the survey on the local database for the questions, and matches answers from the submissions based on `question.body`.  Any answers for other questions that may have been deleted or any changes to the order of the questions are ignored in the export.

One alternative would be to only export survey information that matches if the survey revision matches the one on the submission.  This is the easiest, but makes it harder to compare data if a small change was made to a survey that was resent.

Another alternative would be to allow the user to choose the revision of the survey submissions to export.

The other more complex alternative would be to export in the following format:

 User | Date | Version | Survey Title | Question | Answer 
 ------ | ----- | ----- | ----- | ----- | -----
 as is | as is | The digits of _rev before - | Title of the survey | Question Text | Answer Text

This will leave it up to the users to aggregate to see the survey answers side by side in columns.

**Comment by paulbert** [comment]
Decided in meeting that if a Survey has had submissions sent out we should only allow a copy to be made of a survey, no modifications of that survey.

**Comment by paulbert** [comment]
When opening the **Edit Survey** for a survey that already has responses, the title should have ` - COPY` appended to the end and **Save** should create a new survey in the `exams` database instead of updating.

**Comment by jsschf** [comment]
For me: for the case of a survey being updated more than once, the title should continue to have `- COPY` appended. So a survey that was updated twice would say `My survey - COPY - COPY` and also create a new survey in the `exams` database.

**Comment by jsschf** [comment]
@paulbert @lmmrssa I've spent a lot of time this past week trying to familiarize myself with the codebase and understand how the how the different services work with the different databases. I think I've figured out at a high level what needs to be done but I'm stuck on the actual implementation. 

I would appreciate help or someone walking me through portions of the code I'm having trouble understanding.
